developer/src/tike/main/UfrmHelp.pas

```diff
@@ -55,52 +68,35 @@ implementation
   System.Types,
   Winapi.ShlObj,
 
-  Keyman.Developer.System.HelpTopics,
+  uCEFApplication,
 
+  Keyman.Developer.System.HelpTopics,
+  KeymanDeveloperOptions,
+  Keyman.System.KeymanSentryClient,
   RedistFiles,
   RegistryKeys,
-  uCEFApplication,
   UframeTextEditor,
   UfrmMain,
   UmodWebHTTPServer,
   utilsystem;
 
 procedure TfrmHelp.AddUnmatchedContext(FormName, ControlName: string);
-var
-  i: Integer;
-  n: IXMLNode;
 begin
-  for i := 0 to FHMFRoot.ChildNodes.Count - 1 do
-    if (FHMFRoot.ChildNodes[i].Attributes['FormName'] = FormName) and
-      (FHMFRoot.ChildNodes[i].Attributes['ControlName'] = ControlName) then
-    begin
-      FHMFRoot.ChildNodes[i].Attributes['LastVisited'] := Now;
-      if FHMFRoot.ChildNodes[i].AttributeNodes.IndexOf('VisitCount') < 0 then
-        FHMFRoot.ChildNodes[i].Attributes['VisitCount'] := 1
-      else
-        FHMFRoot.ChildNodes[i].Attributes['VisitCount'] := FHMFRoot.ChildNodes
-          [i].Attributes['VisitCount'] + 1;
-      Exit;
-    end;
+  if not FKeymanDeveloperOptions.ReportUsage then Exit;
+  if TKeymanSentryClient.Instance = nil then Exit;
+  if TKeymanSentryClient.Instance.Client = nil then Exit;
 
-  n := FHMFRoot.AddChild('MissingTopic');
-  n.Attributes['FormName'] := FormName;
-  n.Attributes['ControlName'] := ControlName;
-  n.Attributes['LastVisited'] := Now;
-  n.Attributes['VisitCount'] := 1;
-end;
+  if FormName = '' then FormName := '<empty>';
+  if ControlName = '' then ControlName := '<empty>';
```

Comment:
Are these tags closed automatically? (e.g. `</empty>`)

Reply:
It's not actually a tag in this case, it's just placeholder text (can't have an empty string for Sentry's tags)